### ISSUE TITLE
“LangSwitch” can’t be opened because Apple cannot check it for malicious software.

### ISSUE THREAD

**Issue by Vitalina0** [Nikeev/LangSwitch#6]
"This software needs to be updated. Contact the developer for more information."

What could be done here?

**Comment by Nikeev** [comment]
Hello! Yes, I don't have an Apple Developer certificate, so Apple displays such a notice. 

You could try running the app using the following guide: https://support.apple.com/en-gb/guide/mac-help/mh40616/mac

**Comment by romanr** [comment]
Yes, go to `Settings` → `Privacy & Security` and scroll down to allow.
